x-pack/plugin/searchable-snapshots/src/main/java/org/elasticsearch/xpack/searchablesnapshots/store/SearchableSnapshotDirectory.java

```diff
@@ -356,7 +356,6 @@ public final void close() {
     public void clearCache() {
```

Comment:
I'd change this to `clearCaches(boolean cold, boolean frozen)` and uses it also in tests

Reply:
good idea, I've changed it according to your suggestion.